docusaurus.config.js

```diff
@@ -81,94 +82,108 @@ const config = {
-            position: 'right',
-            className: 'navbar-icon',
+            position: "right",
+            className: "navbar-icon",
           },
           {
             href: discordUrl,
-            'aria-label': 'Discord',
+            "aria-label": "Discord",
             html: '<i class="fa-brands fa-discord"></i>',
-            position: 'right',
-            className: 'navbar-icon',
+            position: "right",
+            className: "navbar-icon",
           },
           {
             href: twitterUrl,
-            'aria-label': 'Twitter / X',
+            "aria-label": "Twitter / X",
             html: '<i class="fa-brands fa-x-twitter"></i>',
-            position: 'right',
-            className: 'navbar-icon',
+            position: "right",
+            className: "navbar-icon",
           },
           {
             href: youtubeUrl,
-            'aria-label': 'YouTube',
+            "aria-label": "YouTube",
             html: '<i class="fa-brands fa-youtube"></i>',
-            position: 'right',
-            className: 'navbar-icon',
+            position: "right",
+            className: "navbar-icon",
           },
         ],
       },
       footer: {
-        style: 'dark',
+        style: "dark",
         links: [
           {
-            title: 'Community',
+            title: "Community",
             items: [
               {
-                label: 'Discord',
+                label: "Discord",
                 href: discordUrl,
               },
               {
-                label: 'Twitter / X',
+                label: "Twitter / X",
                 href: twitterUrl,
               },
             ],
           },
           {
-            title: 'More',
+            title: "More",
             items: [
               {
-                label: 'GitHub',
+                label: "GitHub",
                 href: gitHubUrl,
               },
               {
-                label: 'YouTube',
+                label: "YouTube",
                 href: youtubeUrl,
               },
+            ],
+          },
+          {
+            title: "Contact Us",
+            items: [
               {
-                label:'Email',
-                href:`mailto:${unscramble(obfuscatedEmail, key)}`
-              }
+                html: `
+                      <div>
+                        <a id="emailLink" href="#" target="_blank" rel="noopener noreferrer" class="footer__link-item" onclick="unscrambleEmail(event)">
+                          Email
+                          <svg width="13.5" height="13.5" aria-hidden="true" viewBox="0 0 24 24" class="iconExternalLink">
+                            <path fill="currentColor" d="M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z"></path>
+                          </svg>
+                        </a>
+                      </div>
+                    `,
+              },
```

Comment:
Let's try setting `href: "#contact-email"` to all contact email links (footer and later navigation bar) instead of using raw HTML. Then we could select those elements with `document.querySelectorAll('a[href="#contact-email"]')` and attach the `onclick` event handler to them in a script or [client module](https://docusaurus.io/docs/api/docusaurus-config#clientModules).